src/Species_To_Pokdex_Table.c

```diff
@@ -1220,6 +1220,7 @@ const u16 gSpeciesToNationalPokedexNum[NUM_SPECIES - 1] =
 	// [SPECIES_SLOWPOKE_G - 1] = NATIONAL_DEX_SLOWPOKE,
 	// [SPECIES_SLOWBRO_G - 1] = NATIONAL_DEX_SLOWBRO,
 	[SPECIES_FARFETCHD_G - 1] = NATIONAL_DEX_FARFETCHD,
+	// [SPECIES_KOFFING_G - 1] = NATIONAL_DEX_KOFFING,
```

Comment:
added this in only to comment it out?

Reply:
Right.  The addition was from upstream, and I commented it out because Koffing isn't in our game.  I could remove it I suppose, don't remember the reason we chose to comment out data here in the first place